app/views/admin/tenants/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div id="categories_header">
+  <h1>Borrowers</h1>
+</div>
+<% @tenants.each do |tenant| %>
```

Comment:
I think this would be good to have in a table, one row per tenant with organization and status (eventually we'll add links/buttons to take offline).